AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AssertImageNotFoundAfterSearchFirstSymbolsActionGroup.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertImageNotFoundAfterSearchFirstSymbols">
+        <arguments>
+            <argument name="src" type="string" defaultValue=""/>
+        </arguments>
+        <dontSeeElement selector="//div[@class='masonry-image-column' and contains(@data-repeat-index, '0')]//img[@src='{{src}}']" stepKey="checkFirstPopularImageNotFoundAfterSearchFirstSymbols"/>
+        <dontSeeElement selector="//aside[contains(@class, 'modal-popup')]//div[@class='modal-content']//div[text()='Something went wrong.']" stepKey="errorModalIsAbsentAfterSearchFirstSymbols"/>
+        <dontSeeElement selector="//div[contains(@id, 'adobe-stock-images-search-modal')]//div[@class='spinner']" stepKey="spinnerIsAbsentAfterSearchFirstSymbols"/>
```

Comment:
all of the selectors should be in a section file

Reply:
ok, thanks for some reason i forget for this.